item-api/src/test/java/com/example/itemapi/acceptance/domain/ItemAcceptanceTest.java

```diff
@@ -69,4 +75,53 @@ public class ItemAcceptanceTest extends AcceptanceTest {
                 () -> assertThat(response.jsonPath().getInt("stock")).isEqualTo(0)
         );
     }
+
+    @Test
+    void 재고_차감_웨이브_테스트_5번_웨이브_1000명_동시_요청() throws InterruptedException {
+        ItemRequest.AddItem request = ItemRequest.AddItem.builder()
+                .name("모자")
+                .price(BigDecimal.valueOf(10_000))
+                .stock(10_000)
+                .build();
+        ExtractableResponse<Response> addResponse = 제품_등록_요청(request);
+
+        int waveCount = 5;
+        int concurrencyPerWave = 1_000;
+        decreaseTestWave((no) -> 재고_차감_요청(1L, 1), waveCount, concurrencyPerWave);
+    }
+
+    private void decreaseTestWave(Consumer<Void> action, int waveCount, int concurrencyPerWave) throws InterruptedException {
```

Comment:
```suggestion
    private void decreaseTestWave(Runnable action, int waveCount, int concurrencyPerWave) throws InterruptedException {
```

```
        decreaseTestWave(() -> 재고_차감_요청(1L, 1), waveCount, concurrencyPerWave);

```
이렇게도 개선이 될거같긴한데 한번 확인 부탁드립니다 ㅎㅎ

Reply:
넵! 개선하겠습니다!